platform/android/export/export.cpp

```diff
@@ -847,6 +847,135 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 					uint32_t name = decode_uint32(&p_manifest[iofs + 12]);
 					String tname = string_table[name];
 
+					int dof_index = p_preset->get("graphics/degrees_of_freedom"); // 0: none, 1: 3dof and 6dof, 2: 6dof
+
+					if (tname == "uses-feature" && dof_index > 0) {
+						ofs += 24; // skip over end tag
```

Comment:
Can the logic be extended similar to what's done with the permissions, so that we can have a list of `uses-features` tag that would be iterated through and added as specified by the user.

Reply:
I'm not exactly sure how this could be done as the 6dof option and the 3dof plus 6dof option are mutually exclusive.

Reply:
I was referring to other features besides 3dof/6dof that also necessitate the insertion of the `uses-features` tag.
For example, the [ARCore PR](https://github.com/godotengine/godot/pull/26221/files#diff-b39419f9be37c6f37a62e693300244d1R19) needs to include the `android.hardware.camera.ar` `uses-features` tag.

Feel free to punt on this though since it won't be needed until that PR is ready to be reviewed, at which time, we can expand on the current insertion logic.

Reply:
I'm working on this feature now.

Reply:
Given that to add a general purpose features list would require changing some engine code that I cannot find currently where the Tree is constructed with checkboxes for features to allow setting android version and GL version etc, and the manifest line for the oculus quest is highly customized, this should be punted until that is done in a separate PR. As of right now, oculus quest support along with non 6dof oculus devices like the oculus go is completely broken without this modification, so it should be merged now.

Reply:
Sounds good to me. Once you squashed the commits, it should be good to go.
Great work by the way!